src/v/pandaproxy/schema_registry/service.h

```diff
@@ -54,6 +54,7 @@ class service : public ss::peering_sharded_service<service> {
     sharded_store& schema_store() { return _store; }
     request_authenticator& authenticator() { return _auth; }
     ss::future<> mitigate_error(std::exception_ptr);
+    ss::future<> ensure_started() { return _ensure_started().discard_result(); }
```

Comment:
Should I add the extra commit to remove the result from the future everywhere?

Reply:
i'm not sure i understand the question. if the return types are compatible then isn't that what matters?

Reply:
Sorry, what I mean is that the return type from the oneshot does not need to include units. The units are sufficiently held within the oneshot class and once they have been acquired there is no need to keep them externally to the one shot class. 

I'm asking if Ben thinks I should clean that up now and remove the return type everywhere or if it's fine to just discard the result here in this one place.

Reply:
> I'm asking if Ben thinks I should clean that up now and remove the return type everywhere or if it's fine to just discard the result here in this one place. 

If you need to make changes to this PR anyway, that would be a nice commit, but if this is already good to go, we can do it as a follow up.

Reply:
This is good to go, I'll open another PR for that